Rakefile

```diff
@@ -73,13 +73,15 @@ def open_simulator_and_sleep(platform)
   sh "xcrun instruments -w '#{device_names[platform]} (#{device_os[platform]})' || sleep 15"
 end
 
-def xcodebuild(tasks, platform, xcprety_args: '')
+def xcodebuild(tasks, platform, xcprety_args: '', xcode_summary: false)
   sdk = sdks[platform]
   scheme = schemes[platform]
   destination = devices[platform]
 
   open_simulator_and_sleep(platform)
-  safe_sh "set -o pipefail && xcodebuild -project '#{moya_project}' -scheme '#{scheme}' -configuration '#{configuration}' -sdk #{sdk} -destination '#{destination}' #{tasks} | bundle exec xcpretty -c #{xcprety_args}"
+  xcpretty_json_output_name = xcode_summary == true ? " XCPRETTY_JSON_FILE_OUTPUT=\"xcodebuild-#{platform}.json\"" : ""
+  xcpretty_formatter = xcode_summary == true ? " -f `bundle exec xcpretty-json-formatter`" : ""
+  safe_sh "set -o pipefail && xcodebuild -project '#{moya_project}' -scheme '#{scheme}' -configuration '#{configuration}' -sdk #{sdk} -destination '#{destination}' #{tasks} |#{xcpretty_json_output_name} bundle exec xcpretty -c #{xcprety_args}#{xcpretty_formatter}"
```

Comment:
Does this need a space after the `|`? There isn't any at this moment.

Reply:
I'm not sure but even if it's not needed is good to have separation :)